src/test/scala/uk/gov/hmrc/stubs/TestDataFactory.scala

```diff
@@ -71,6 +98,21 @@ object TestDataFactory {
     order = First
   )
 
+  def validSeniorAccountingOfficer(
+    fullName: String = Defaults.contactName
+  ): SeniorAccountingOfficer = SeniorAccountingOfficer(
+    fullName = fullName,
+    email = emailFor(fullName),
+    accountingPeriod = validSeniorAccountingOfficerPeriod()
+  )
+
+  def validPastSeniorAccountingOfficer(
+    fullName: String = Defaults.saoName
+  ): PastSeniorAccountingOfficer = PastSeniorAccountingOfficer(
+    fullName = fullName,
+    accountingPeriod = validPastSeniorAccountingOfficerPeriod()
```

Comment:
Similar to above, I reckon you could probably just remove validSeniorAccountingOfficerPeriod(), and do this:

```suggestion
    accountingPeriod = validAccountingPeriod().copy(startDate = oneYearAgo , dueDate = None)
```

Reply:
validPastSeniorAccountingOfficerPeriod removed and amended based on code provided